src/regions/ca/i18n/steps.fr.ts

```diff
@@ -30,11 +30,11 @@ export default {
   yes: 'Oui',
   no: 'Non',
   askForLocation:
-    'Dans quelle province ou territoire êtes-vous actuellement situé?',
+    'Dans quelle province ou territoire êtes-vous actuellement situé ?',
```

Comment:
@hollyorlando ouh.... bold... joke :D

Reply:
@vanessajones hahha, I figured we should stick to something, and there were more with spaces than without 🙈

Reply:
hahaha definitely - way to lead the way! 💪